UpdateProfile/handler.ts

```diff
@@ -97,10 +99,36 @@ export function UpdateProfileHandler(
       profilePayload.email !== undefined &&
       profilePayload.email !== existingProfile.email;
 
+
+    // Get servicePreferencesSettings mode from payload or default to LEGACY
+    // tslint:disable-next-line: prettier
+    const requestedServicePreferencesSettingsMode = profilePayload.service_preferences_settings?.mode ?? ServicesPreferencesModeEnum.LEGACY;
+
+    // Check if a mode change is requested
+    const servicePreferencesSettingsModeChanged = requestedServicePreferencesSettingsMode !== existingProfile.servicePreferencesSettings.mode;
```

Comment:
Can we use `is` prefix for constants and functions that are or return boolean values?
```suggestion
    const isServicePreferencesSettingsModeChanged = requestedServicePreferencesSettingsMode !== existingProfile.servicePreferencesSettings.mode;
```

Reply:
fixed in 01360d9